webapp/src/pages/FoodCheckout.jsx

```diff
@@ -148,26 +147,30 @@ export default function FoodCheckout() {
             <hr />
           </>
         )}
-        <LayoutContainer gutters className="mb-4 mt-4">
-          <CheckoutFulfillment
-            checkout={checkout}
-            showErrorToast={showErrorToast}
-            register={register}
-            errors={errors}
-            onCheckoutChange={(attrs) =>
-              runSetter("fulfillmentOption", setCheckoutMutations, {
-                ...checkoutMutations,
-                ...attrs,
-              })
-            }
-          />
-        </LayoutContainer>
-        <hr />
-        <LayoutContainer gutters className="mb-4 mt-4">
+        {!isEmpty(checkout.availableFulfillmentOptions) && (
+          <>
+            <LayoutContainer gutters className="my-4">
```

Comment:
I don't understand why we are using nested containers like this. It's a bad practice, as per the bootstrap docs: https://getbootstrap.com/docs/5.0/layout/containers/

If something is called 'layout' you normally only have one of them.

This is too big of an issue to fix now but please do not continue using LayoutContainer (or nested Containers) everywhere. If you want flexbox, just use flexbox. If you want a grid, use bootstrap row/col. I think the changes here, removing all the grid components, are largely a step backwards, but 🤷